deploy/README.adoc

```diff
@@ -32,7 +32,6 @@ Also, there may be newer versions of the components I used, maybe you can upgrad
 
 The usual command line tools. I would like to call out the following:
 
-* `tkn` - The Tekton command line client. You can ignore it if you don't want to use Tekton.
```

Comment:
I am hesitating to remove Tekon. Cross compiling Rust can be tricky. And Tekton takes away the pain of that for Mac and Windows users.

@dejanb had some troubles with that.

Maybe the solution is a local container build, rather than using Tekton.

Reply:
I did not remove it ! It was just moved into a separate file....


.....


... which i forgot to commit :facepalm: